performance/src/test/com/clickhouse/benchmark/clients/BenchmarkBase.java

```diff
@@ -135,12 +158,14 @@ public static void insertData(String tableName, InputStream dataStream, ClickHou
 
     public static void verifyRowsInsertedAndCleanup(DataSet dataSet) {
         try {
+            syncQuery(dataSet);
```

Comment:
is there anything we can do to avoid manual call? it's easy to forget to add one